README.md

```diff
@@ -42,44 +42,61 @@ Fork the repository so that you can create and work with branches independently
 After you have forked the repository, you must clone it to your local machine and add the original `redhataccess/pantheon` repository as an upstream remote.
 
 1. From a terminal, clone the repository:
+       ```
        $ git clone git@github.com:<user-space>/pantheon.git
+       ```
 1. Set up `redhataccess/pantheon` as the upstream:
+       ```
        $ cd pantheon
        $ git remote add upstream git@github.com:redhataccess/pantheon.git
+       ```
 
 ### Creating a working branch
 
 Whenever you work on a new issue, you must create a new working branch based on the latest version of the upstream master branch.
 
 1. Ensure you are on master
+       ```
        $ git checkout master
+       ```
 1. Ensure your fork is up to date
+       ```
        $ git pull upstream master
+       ```
 1. Create a working branch based on the issue in JIRA:
+       ```
        $ git checkout -b FCCEUD<ID#>
+       ```       
 
 ### Creating a pull request and completing review
 
 When your work is ready to be reviewed and merged, create a pull request.
 
 1. Push your working branch to your fork:
+       ```
        $ git push -u origin <branch_name>
+       ```
 1. From the repository page in GitHub, click **New pull request**.
 1. Select your working branch from the compare list.
-1. Add `WIP` to the title of the pull request.
-1. Add the **awaiting tech review** label to the pull request.
-1. In the pull request comment field, enter `@redhataccess/pantheon-developers Please review for technical completeness and accuracy`.
+1. Add `WIP:` to the title of the pull request. This automatically converts the pull request to a draft pull request.
 1. Click **Create new pull request**.
+1. Add the **awaiting tech review** label to the pull request.
+1. In the pull request comment field, enter `@redhataccess/eud-tech-review Please review for technical completeness and accuracy`.
 
-For code pull requests, one or more developers review the pull request. For documentation pull requests, the developers review the pull request for technical accuracy and documentation team members review the pull request for clarity, consistency, and compliance with necessary standards.
 
 ### The review process
 
 Both the technical review and peer review processes take place in pull requests in GitHub.
 
+#### Documentation review
 After creating and labeling a pull request as outlined above, the developers review the pull request and add comments regarding technical accuracy. Writers receive a notification that comments have been added via email, and when all comments have been addressed, the developers change the label from **awaiting tech review** to **tech review passed**.
```

Comment:
```suggestion
After creating and labeling a pull request as outlined above, the developers review the pull request and add comments regarding technical accuracy. Writers receive a notification that comments have been added via email, and when all comments have been addressed, the developers change the label from **awaiting tech review** to **tech review complete**.
```

Reply:
@dvagnero  Yes, you're right, I made the suggested change. Please give approval again.